cypress/e2e/WebInterface/Measure/CreateMeasure/CreateMeasureValidations.cy.ts

```diff
@@ -1049,11 +1049,11 @@ describe('Measure Creation: CV ListQDMPositiveEncounterPerformed With MO And Str
             .then((radio) => {
                 //confirm that initial value is set to 'No'
                 cy.wrap(radio).eq(1).should('be.checked');
-                cy.contains('[class="MuiFormControlLabel-root MuiFormControlLabel-labelPlacementEnd css-1jaw3da"]', 'No');
+                cy.pause()
```

Comment:
pause here

Reply:
Doh!
I can't believe I left that in there! LOL
I just removed this, so it _should_ be good-to-go, now.